### PR TITLE
Implement data aquisition

### PR DESCRIPTION
**Pull request by nstephenh**
Implementing #25

- [x] Load list from gallery
- [x] Create workspace from selected cat
- [ ] Create workspace from playground when it's selected.

Draft PR as I still have to create the workspace and implement something for playground.

### CODE REVIEW

File: src/Phalanx.App/Pages/Index.razor [hunk 1]
```diff
@@ -1,8 +1,68 @@
...
+<FluentSelect @bind-Value="selectedGalleryName">
+    <FluentOption>Use Playground</FluentOption>
+
+    @foreach (var galleryRef in state.Cache.GalleryReferences)
+    {
+        <FluentOption Value="galleryRef.Name">@galleryRef.Name</FluentOption>
+    }
+
+</FluentSelect>
+<Phalanx.App.Pages.Configuration.SystemSelect @bind-CatpkgGallery=SelectedGalleryCache></Phalanx.App.Pages.Configuration.SystemSelect>
```
[the remark this answers is not in the document]
Comment (nstephenh) [reply src/Phalanx.App/Pages/Index.razor]: Done. Moved to "/rostercreateselectsystem"

**Comment by nstephenh** [comment]
Whoops, have to push first :P. I'm used to having a "commit and push" button.

File: src/Phalanx.App/Phalanx.App.csproj [hunk 2]
```diff
@@ -9,21 +9,30 @@
...
   <ItemGroup>
     <ProjectReference Include="..\Phalanx.SampleDataset\Phalanx.SampleDataset.csproj" />
+    <ProjectReference Include="..\Warhub.ArmoryModel.DataProviders\Warhub.ArmoryModel.DataProviders.csproj" />
     <ProjectReference Include="..\WarHub.ArmouryModel.Concrete.Extensions\WarHub.ArmouryModel.Concrete.Extensions.csproj" />
     <ProjectReference Include="..\WarHub.ArmouryModel.EditorServices\WarHub.ArmouryModel.EditorServices.csproj" />
     <ProjectReference Include="..\WarHub.ArmouryModel.Extensions\WarHub.ArmouryModel.Extensions.csproj" />
   </ItemGroup>
 
   <ItemGroup>
-    <!-- <ServiceWorker Include="wwwroot\service-worker.js" PublishedContent="wwwroot\service-worker.published.js" /> -->
```
Comment (amis92) [inline src/Phalanx.App/Phalanx.App.csproj:26]: I don't think these changes are necessary. `wwwroot/appsettings.json` will automatically be included.

**Review by amis92 — APPROVED** [review]

**Comment by nstephenh** [comment]
We decided to skip creating the workspace from playground for now, until we have a workflow to show the workspace and integrate this with.

**Pull request merged**